### PR TITLE
fix an ambiguous question and format the code blocks

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/browser-apis/toolbox-ii/network-information-api.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/browser-apis/toolbox-ii/network-information-api.md?ref=a5a546ef1452131dc9a179a5475770d931ece3d8)

File: javascript/browser-apis/toolbox-ii/network-information-api.md [hunk 1]
```diff
@@ -17,14 +9,10 @@ aspects:
   - introduction
   - new
   - workout
-
-standards:
-  javascript.browser-apis-device.2: 10
+  
 links:
-
   - '[code.tutsplus.com](http://code.tutsplus.com/tutorials/html5-network-information-api--cms-21598){website}'
```
Comment (kapnobatai137) [inline javascript/browser-apis/toolbox-ii/network-information-api.md]:
```suggestion
  - '[Network Information API](http://code.tutsplus.com/tutorials/html5-network-information-api--cms-21598){website}'
```

File: javascript/browser-apis/toolbox-ii/network-information-api.md [hunk 2]
```diff
@@ -39,22 +27,35 @@ The API has a *`NetworkInformation`* interface and a single property to the Navi
...
 ```javascript
-var connection = navigator.connection ||
-              navigator.mozConnection ||
-              navigator.webkitConnection;
-var type = connection.type;
+let connection =
+  navigator.connection ||
+  navigator.mozConnection ||
+  navigator.webkitConnection;
+let type = connection.type;
```
Comment (kapnobatai137) [inline javascript/browser-apis/toolbox-ii/network-information-api.md]: https://developer.mozilla.org/en-US/docs/Web/API/Network_Information_API

image

```suggestion
let type = connection.effectiveType;
```

File: javascript/browser-apis/toolbox-ii/network-information-api.md [hunk 3]
```diff
@@ -64,32 +65,35 @@ function updateConnectionStatus() {
...
+  navigator.??? ||
+  navigator.webkitConnection;
+let type = connection.???;
 ```
 
-
 * `navigator`
 * `mozConnection`
-* `webkitConnection`
+* `type`
```
Comment (kapnobatai137) [inline javascript/browser-apis/toolbox-ii/network-information-api.md]:
```suggestion
* `effectiveType`
```

File: javascript/browser-apis/toolbox-ii/network-information-api.md [hunk 4]
```diff
@@ -39,22 +27,35 @@ The API has a *`NetworkInformation`* interface and a single property to the Navi
...
 ```javascript
 function updateConnectionStatus() {
-  console.log("Connection type is change \
-  from " + type + " to " + connection.type);
+  console.log(
+    "Connection type is change \
+  from " +
+      type +
+      " to " +
+      connection.type
```
Comment (kapnobatai137) [inline javascript/browser-apis/toolbox-ii/network-information-api.md]:
```suggestion
      connection.effectiveType
```

File: javascript/browser-apis/toolbox-ii/network-information-api.md [hunk 5]
```diff
@@ -39,22 +27,35 @@ The API has a *`NetworkInformation`* interface and a single property to the Navi
...
 ```
+
 Function to output for any change to the user's connection if called by the listener.
+
 ```javascript
 function updateConnectionStatus() {
-  console.log("Connection type is change \
-  from " + type + " to " + connection.type);
+  console.log(
+    "Connection type is change \
```
Comment (kapnobatai137) [inline javascript/browser-apis/toolbox-ii/network-information-api.md]: this should be `changed` right?
```suggestion
    "Connection type is changed \
```

**Comment by kapnobatai137** [comment]
I will also come back to this to add browser compatibility explanation 

image

**Comment by nem035** [comment]
@kapnobatai137 instead of mentioning the compatible or incompatible systems, we can make the content more time-proof by just linking to a compatibility table and add a general warning along the lines of  "Do note that this feature might not be supported on all platforms. Checkout the compatibility table"

**Comment by nem035** [comment]
Good to merge? @kapnobatai137 @Stefan-Stojanovic

**Comment by kapnobatai137** [comment]
> 
> 
> Good to merge? @kapnobatai137 @Stefan-Stojanovic

yes, I didn't get to add browser compatibility, but made a note of it and adding it after merging

**Pull request merged**